ApiReview/API.Polly.Core/NoDocs/Polly.Retry/RetryDelayGeneratorArguments.cs

```diff
@@ -0,0 +1,15 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Polly.Retry;
+
+public readonly struct RetryDelayGeneratorArguments<TResult>
+{
+    public Outcome<TResult> Outcome { get; }
+    public ResilienceContext Context { get; }
+    public int AttemptNumber { get; }
+    public TimeSpan DelayHint { get; }
```

Comment:
what is the purpose of this hint?

Reply:
This value is generated by the retry strategy and respects the the retry options and current attempt number.
If generator is not able to extract the delay from the  outcome it can return this value as a next delay attempt.